services/elasticache/pkg/resource/cache_subnet_group/sdk.go

```diff
@@ -133,6 +133,7 @@ func (rm *resourceManager) sdkCreate(
 	ctx context.Context,
 	r *resource,
 ) (*resource, error) {
+
```

Comment:
Can you remove these two code changes please? I think you may have just added a newline into the sdk template?